doc/whatsnew.rst

```diff
@@ -11,7 +11,7 @@ What's New in PypeIt
 
 ----
 
-.. include:: releases/1.16.1dev.rst
+.. include:: releases/1.17.0.rst
 
```

Comment:
Also add `releases/1.17.1dev.rst`.

Reply:
Thanks - will add this in another PR for completeness.